### PR TITLE
document used paymentmethods.

### PR DESCRIPTION
**Pull request by ahalberkamp**
In this pr the payment-methods, which were provided by payone, were  documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +52,38 @@ It is a standalone Microservice that connects the two platforms and provides a s
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Supported payment methods
+
+The PAYONE Integration Service currently supports the following payment methods:
```
Comment (salander85) [inline README.md:57]: Add a link to payone doc and/or add short descriptions of the payment methods. If there is further information documented e.g. Klarna activation - also link it.

Comment (ahalberkamp) [reply README.md:57]: fab8376eb

File: README.md [hunk 2]
```diff
@@ -16,6 +16,24 @@ It is a standalone Microservice that connects the two platforms and provides a s
...
+  - [Configuration of the Integration Service](#configuration-of-the-integration-service)
+  - [Mandatory common properties](#mandatory-common-properties)
+      - [Mandatory commercetools API client credentials](#mandatory-commercetools-api-client-credentials)
+      - [PAYONE API client credentials](#payone-api-client-credentials)
+      - [Optional service configuration parameters](#optional-service-configuration-parameters)
+    - [Docker run](#docker-run)
+- [Shop integration guide](#shop-integration-guide)
+- [Multitenancy](#multitenancy)
+- [Other resources](#other-resources)
+- [For developers and contributors](#for-developers-and-contributors)
```
Comment (butenkor) [inline README.md:33]: It seems that you have content tree now generated twice? see below...

File: README.md [hunk 3]
```diff
@@ -34,6 +52,51 @@ It is a standalone Microservice that connects the two platforms and provides a s
...
+    Available in: Suisse
+  * [PostFinance Card](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+Postfinance)  
+    Available in: Suisse
+  * [iDEAL](https://docs.payone.com/display/public/PLATFORM/Special+Remarks+-+Recurring+Transactions+with+iDeal)  
+    Available in: Netherlands
+  * Bancontact  
+    Available in: Belgium 
+      
+* Financing  
+  * [Klarna](https://github.com/commercetools/commercetools-payone-integration/blob/master/docs/Klarna-Activation.md)
```
Comment (butenkor) [inline README.md:97]: Would be nice to link to our payment specific integration docs also for other methods.

Comment (ahalberkamp) [reply README.md:97]: we only have a internal documentation for klarna :-)....I have double checked it

File: README.md (hunk 2 above)
Comment (ahalberkamp) [reply README.md:33]: 9869bbaf0

File: README.md [hunk 4]
```diff
@@ -31,9 +32,53 @@ It is a standalone Microservice that connects the two platforms and provides a s
...
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Supported payment methods
+
+The PAYONE Integration Service currently supports the following payment methods:
+
+* Direct debit
+  Available in: worldwide
+* Advance payment  
+  Available in: Europa
```
Comment (salander85) [inline README.md:44]:
```suggestion
  Available in: Europe
```

File: README.md (hunk 3 above)
Comment (salander85) [reply README.md:79]: Wouldn't it be nice to have integration docs for all methods? WDYT @butenkor

Comment (ahalberkamp) [reply README.md:79]: A lot payment-methods has the same workflow and for each "special" step in the workflow, I have linked the related payone-documentation, so I think that is enought

**Review by salander85 — APPROVED** [review]
LGTM! 👍

**Review by butenkor — APPROVED** [review]

**Pull request merged**